oshi-core/src/main/resources/oshi.properties

```diff
@@ -267,3 +267,7 @@ oshi.os.openbsd.filesystem.path.excludes=/tmp**,/dev
 
 oshi.os.solaris.filesystem.path.excludes=/system**,/tmp**,/dev,/dev/fd**
 oshi.os.solaris.filesystem.volume.excludes=rpool
+
+# List of types inside /sys/class/thermal/thermal_zone*/type to detect as CPU temperature.
+# The order of entries in the list does represent the priority, the first match will be used.
+oshi.os.linux.sensors.cpuTemperature.types=cpu-thermal,x86_pkg_temp
```

Comment:
Nit: please add a final line break to end the file.  Apparently we didn't include the `.properties` extension in our `pom.xml` spotless configuration :)